config/country_codes.yml

```diff
@@ -1,5 +1,253 @@
 ---
 iso:
```

Comment:
Please add a comment with the source of  this information in case we need to update this file in the future:

Either [this](http://data.okfn.org/data/core/country-list) or [this](https://raw.githubusercontent.com/datasets/country-list/master/data.csv) link.